src/python-common/setup.py

```diff
@@ -12,6 +12,9 @@
     keywords='ceph',
     url="https://github.com/ceph/ceph",
     zip_safe = False,
+    install_requires=(
```

Comment:
this change belongs to bc1370c902ef36018589e23cf96ac2d14b3aa787

Reply:
fixed